blueprints/app/files/.stylelintignore

```diff
@@ -0,0 +1,12 @@
+# unconventional files
+/blueprints/*/files/
+
+# compiled output
+/dist/
+
+# misc
+!.*
```

Comment:
Why did we need this again?

Reply:
Usually this is for un-ignoring hidden files. Removed since I don't imagine hidden CSS files are common.